Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -331,8 +331,34 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
             if did_remove:
                 pr_issue.set_labels(labels)
 
+        commits = list(repository.commits(begin=dict(hash=branch_point.hash), end=dict(branch=repository.branch)))
+
+        issue = None
+        for line in commits[0].message.split() if commits[0] and commits[0].message else []:
+            issue = Tracker.from_string(line)
+            if issue:
+                break
+
         if isinstance(remote_repo, remote.GitHub):
+            if issue and issue.redacted and args.remote is None: # TODO: is this a correct check on args.remote?
+                print("Your issue is redacted, which means it will be diverted to a secure, non-origin remote you have access to.")
+                if args.defaults or Terminal.choose(
+                    "Would you like to override this behavior? \n",
+                    default='No',
+                ) == 'No':
+                    original_remote = source_remote
+                    if len(repository.source_remotes()) < 2:
+                        print("Error. You do not have access to a secure, non-origin remote")
```

Comment:
Should send this to `sys.stderr` instead of printing it.

Reply:
Should this one go to stderr, or the following one that actually cancels it? This one prompts them to reconsider overriding and then they can select

Reply:
This one. Basically, the stuff that explains why you're returning an error code.

Reply:
Should I just not even ask them if they want to proceed anyways? Just go directly to stderr, no further Terminal interaction?

Reply:
Oh, I'm sorry, I misread this. I still think this should be `sys.stderr`, since you're starting the line with `error`, but it's not as clear cut as I originally said, sorry.